py/desisim/bal.py

```diff
@@ -45,10 +45,28 @@ def empty_balmeta(self, qsoredshift=None):
         if qsoredshift is not None:
             balmeta['REDSHIFT'] = qsoredshift
 
-        return balmeta        
+        return balmeta
+
+    #Since we want to store most of the balmeta table it seemed better to copy the template balmeta and complement it with the missing columns
+    #And remove those that are not used...
+    def template_balmeta(self,indx):
+        """Initialize an empty metadata table for BALs."""
+
+        from astropy.table import Table, Column
+        from desisim.io import find_basis_template
+
+        balmeta=self.balmeta.copy()
+        balmeta.remove_columns(['SDSS_NAME', 'RA', 'DEC', 'PLATE', 'MJD', 'FIBERID'])
+        balmeta=balmeta[indx]
+        nbal=len(balmeta)
+        balmeta.add_column(Column(name='BAL_TEMPLATEID',length=nbal, dtype='i4', data=np.zeros(nbal)-1),index=0)
+        balmeta.add_column(Column(name='BALPROB',length=nbal, dtype='f4', data=np.ones(nbal)),index=0)
+        balmeta.add_column(Column(name='Z',length=nbal, dtype='f4', data=np.zeros(nbal)),index=0)
+        balmeta.add_column(Column(name='TARGETID',length=nbal, dtype='i4', data=np.zeros(nbal)-1),index=0)
+        return balmeta
         
     def insert_bals(self, qsowave, qsoflux, qsoredshift, balprob=0.12,
-                    seed=None, verbose=False):
+                    seed=None, verbose=False,qsoid=None):
         """Probabilistically inserts BALs into one or more QSO spectra.
```

Comment:
Very minor comment, but we've made an effort to conform to the [`PEP8`](https://www.python.org/dev/peps/pep-0008/) style guide. It would be nice if the proposed changes at least matched the surrounding style.

Reply:
### Ok I've fixed it and some simmilar ones. I'll try to follow the style guide. Thanks